robot-core/src/test/java/org/obolibrary/robot/IOHelperTest.java

```diff
@@ -324,7 +324,15 @@ public void testStrict() throws IOException {
     IOHelper ioHelper = new IOHelper();
     ioHelper.setStrict(true);
     String input =
-        "@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .\n\n_:Bb65616 rdf:type rdf:Statement .";
+        "@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> ."
+            .concat("\n\n")
+            .concat("@prefix foo: <http://example.com#> .")
+            .concat("\n\n")
+            .concat("_:Bb65616 rdf:type rdf:Statement ;")
+            .concat("\n\n")
+            .concat("rdf:object foo:Bar ;")
+            .concat("\n\n")
+            .concat("rdf:subject foo:Foo .");
```

Comment:
Essential change?

Reply:
Yes this is essential. Only `_:Bb65616 rdf:type rdf:Statement .` triple is not causing unparsed triples in the owlapi 4.5.22. `rdf:object` and `rdf:subject`statements  are generating the expected exception.

Reply:
Thank you for the clarification.